backend/app/models/user.rb

```diff
@@ -41,6 +41,10 @@
 class User < ApplicationRecord
   enum role: { contributor: 0, publisher: 1, admin: 2 }
 
+  has_many :participant, dependent: :destroy
```

Comment:
"has_many" relationships should always be pluralized: has_many :participants , belongs_to are singular. I know that sometimes it doesn't read so well from one of the directions, like in this case where the user is the participant... but probably my fault on the naming!